app/controllers/reports_controller.rb

```diff
@@ -2,8 +2,21 @@
 class ReportsController < ApplicationController
   def create
     @info_request = InfoRequest.find_by_url_title!(params[:request_id])
+    if params[:comment_id]
```

Comment:
2 broad approaches to consider...

1) "just" send an email about the comment (or message) to ask admins to look (but how will this be reflected admin interface?)
2) make comments reportable, possibly via Concerns so that we can extend it to other objects more readily, to store their own data about having been reported, again will require something new for the admin interface to be genuinely useful